src/test/ui/suggestions/suggest-let-for-assignment.stderr

```diff
@@ -0,0 +1,33 @@
+error[E0425]: cannot find value `x` in this scope
+  --> $DIR/suggest-let-for-assignment.rs:4:5
+   |
+LL |     x = 1;
+   |     ^ help: you might have meant to introduce a new binding: `let`
+
```

Comment:
@estebank 
I don't understand why my current code can not show suggestions like this now:

```
help: you might have meant to introduce a new binding + yukang
  |
4 |     let x = 1; //~ ERROR cannot find value `x` in this scope
  |     +++
```

The current code is:
```rust
 err.span_suggestion(
                ident_span.shrink_to_lo(),
                "you might have meant to introduce a new binding",
                "let ".to_string(),
                Applicability::MachineApplicable,
            );
```

If I change the `msg` to some different one, it will show as above.

Reply:
Got it, `span_suggestion_verbose` will looks better.